commnad_line_rock_paper_scissors.py

```diff
@@ -6,6 +6,27 @@
 from rock_paper_scissors.game import Game
 
 
+def human_input_to_move(game, human):
+    choices = ', or '.join(
+        [
+            ', '.join(game.valid_choices[:-1]),
+            game.valid_choices[-1]
+        ]
+    )
+
+    try:
+        human.choice = game.convert_input_to_move(input('Choose {}: '.format(choices)))
```

Comment:
I'm not sure how to catch errors that are rooted in the User and Game class other than returning the function with the params at the end of the `except` clause 
```
     try:
        return game.convert_input_to_move(input('Choose {}: '.format(choices)))
    except Exception:
        print('Input from {} is invalid. Enter another choice'.format(human))
        *return human_input_to_move(game, human)*